.drone.env

```diff
@@ -1,6 +1,6 @@
 # The test runner source for API tests
-CORE_COMMITID=b587826832245d9c5f72ab2a9d831b1be6b66210
-CORE_BRANCH=master
+CORE_COMMITID=
+CORE_BRANCH=dontRunOldPublicWebdavInOcis
```

Comment:
**note** needs this to be changed back to master after https://github.com/owncloud/core/pull/38825#pullrequestreview-681685219 this gets merged

Reply:
This is done now.